diff -up firefox-68.5.0/js/src/moz.build.p33~ firefox-68.5.0/js/src/moz.build
--- firefox-68.5.0/js/src/moz.build.p33~	2020-03-11 19:02:39.336831811 +0100
+++ firefox-68.5.0/js/src/moz.build	2020-03-11 19:03:48.130828473 +0100
@@ -428,6 +428,8 @@ DIRS += [
 
 FINAL_LIBRARY = 'js'
 
+USE_LIBS += [ 'mozglue' ]
+
 # Prepare self-hosted JS code for embedding
 GENERATED_FILES += [('selfhosted.out.h', 'selfhosted.js')]
 selfhosted = GENERATED_FILES[('selfhosted.out.h', 'selfhosted.js')]
